pkgs/top-level/python-packages.nix

```diff
@@ -13867,6 +13867,30 @@ in {
     clblas = pkgs.clblas-cuda;
   };
 
+    libnacl = buildPythonPackage rec {
+    name = "libnacl-${version}";
+    version = "1.5.0";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/b8/59/62b7f0952da7ce6d677c8cc78ef62defe48c99bdf5822ca407b5a5c7008f/libnacl-1.5.0.tar.gz";
+      sha256 = "1ph042x0cfysj16mmjif40pxn505rg5c9n94s972dgc0mfgvrwhs";
+    };
+
+    propagatedBuildInputs = with self; [ pkgs.libsodium ];
+
+    patchPhase = ''
+      substituteInPlace "./libnacl/__init__.py" --replace "ctypes.cdll.LoadLibrary('libsodium.so')" "ctypes.cdll.LoadLibrary('${pkgs.libsodium}/lib/libsodium.so')"
+    '';
+    #doCheck = false;
```

Comment:
It seems they are not found, there is no error but it says `Ran 0 tests in 0.000s`. Any tip on how to proceed?